scripts/ci_tests/rust_tests.sh

```diff
@@ -178,6 +178,17 @@ function tests_using_postgres() {
     cargo test --profile simulator --package iota-indexer --test rpc-tests --features shared_test_runtime
 }
 
+function deny() {
+    local MANIFEST_PATH=${MANIFEST_PATH:-"./Cargo.toml"}
+    cargo deny --manifest-path "$MANIFEST_PATH" check bans licenses sources
+    # check security advisories (in-house crates)
+    cargo deny --manifest-path "$MANIFEST_PATH" check advisories
+}
+
+function deny_external() {
+    MANIFEST_PATH="./external-crates/move/Cargo.toml" deny
```

Comment:
does this line call the deny function then? Maybe add a comment, that wasn't clear to me at the first view, and I thought you forgot the "cargo" 😅

Reply:
I'll rename the function to something a bit more distinctive then 👍